Rakefile

```diff
@@ -66,16 +67,46 @@ directory PKGDIR
 directory "#{BUILDDIR}/parser_cache"
 file "#{BUILDDIR}/parser_cache" => BUILDDIR
 
-file "#{BUILDDIR}/foreman.yaml" => 'config/foreman.yaml' do |t|
-  cp t.prerequisites[0], t.name
-  sh 'sed -i "s#\(.*answer_file:\).*#\1 %s#" %s' % ["#{SYSCONFDIR}/foreman-installer/scenarios.d/foreman-answers.yaml", t.name]
-  sh 'sed -i "s#\(.*installer_dir:\).*#\1 %s#" %s' % ["#{DATADIR}/foreman-installer", t.name]
-  sh 'sed -i "s#\(.*log_dir:\).*#\1 %s#" %s' % ["#{LOGDIR}/foreman-installer", t.name]
-  sh 'sed -i "s#\(.*module_dirs:\).*#\1 %s#" %s' % ["#{DATADIR}/foreman-installer/modules", t.name]
-  sh 'sed -i "s#\(.*hiera_config:\).*#\1 %s#" %s' % ["#{DATADIR}/foreman-installer/config/foreman-hiera.conf", t.name]
-  sh 'sed -i "s#\(.*parser_cache_path:\).*#\1 %s#" %s' % ["#{DATADIR}/foreman-installer/parser_cache/foreman.yaml", t.name]
+SCENARIOS.each do |scenario|
+  scenario_config_replacements = {
+    'answer_file' => "#{SYSCONFDIR}/foreman-installer/scenarios.d/#{scenario}-answers.yaml",
+    'hiera_config' => "#{DATADIR}/foreman-installer/config/foreman-hiera.conf",
+    'installer_dir' => "#{DATADIR}/foreman-installer",
+    'log_dir' => "#{LOGDIR}/foreman-installer",
+    'module_dirs' => "#{DATADIR}/foreman-installer/modules",
+    'parser_cache_path' => "#{DATADIR}/foreman-installer/parser_cache/#{scenario}.yaml",
+  }
   if ENV['KAFO_MODULES_DIR']
-    sh 'sed -i "s#.*\(:kafo_modules_dir:\).*#\1 %s#" %s' % [ENV['KAFO_MODULES_DIR'], t.name]
+    scenario_config_replacements['kafo_modules_dir'] = ENV['KAFO_MODULES_DIR']
+  end
+
+  file "#{BUILDDIR}/#{scenario}.yaml" => "config/#{scenario}.yaml" do |t|
+    cp t.prerequisites[0], t.name
+    scenario_config_replacements.each do |setting, value|
+      sh 'sed -i "s#\(.*%s:\).*#\1 %s#" %s' % [setting, value, t.name]
+    end
+  end
+
+  file "#{BUILDDIR}/parser_cache/#{scenario}.yaml" => ["#{BUILDDIR}/modules", "#{BUILDDIR}/parser_cache"] do |t|
+    sh "#{exporter}/kafo-export-params -c config/#{scenario}.yaml -f parsercache --no-parser-cache -o #{BUILDDIR}/parser_cache/#{scenario}.yaml"
+  end
+
+  # TODO: filename per scenario?
```

Comment:
I'm not entirely sure how to handle this. It doesn't really make sense to have the options of one scenario be part of the man page. Should we have a separate man page per scenario besides the generic one?

Reply:
That's tough in part because not every scenario has a 1:1 mapping with parser cache as well in their current form. There is a lot of re-use. Given this is how things currently work perhaps we can leave this as is for now?

Reply:
I think we should have a static `options.asciidoc` with the generic options and then a manpage per scenario.

Reply:
That makes sense if its easy enough to do. I don't think Katello has this today so the current functionality would not be a regression.